src/include/main/db_main.h

```diff
@@ -714,6 +722,25 @@ class DBMain {
       return *this;
     }
 
+    /**
+     *
+     * @param value with pilot
+     * @return  self reference for chaining
```

Comment:
slightly weird spacing, either have both param and return tab aligned or just use one space